server/ladder_service.py

```diff
@@ -125,8 +126,26 @@ async def fetch_map_pools(self, conn) -> Dict[int, Tuple[str, List[Map]]]:
             _, map_list = map_pool_maps[id_]
             if row.map_id is not None:
                 map_list.append(
-                    Map(row.map_id, row.display_name, row.filename)
+                    Map(row.map_id, row.display_name, row.filename, row.weight)
                 )
+            elif row.map_params is not None:
+                try:
+                    params = json.loads(row.map_params)
+                    map_type = params["type"]
+                    if map_type == "neroxis":
+                        map_list.append(NeroxisGeneratedMap.of(params=params, weight=row.weight))
+                    else:
+                        self._logger.warning("Unsupported map type %s with parameters %s",
+                                             map_type, row.map_params)
```

Comment:
Also I'm not sure if its actually useful to log the map params here. Maybe the id would be better since if I see this warning I probably just want to be able to find that row in the database and fix it/remove it.

Reply:
What format do you prefer?

Reply:
oh never mind I see it